### PR TITLE
Backport Errorprone Config Changes for JDK 16

### PR DESCRIPTION
**Pull request by meltsufin**
An attempt to fix JDK 16 build due to errorprone failing.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -392,26 +392,65 @@
...
+					</plugins>
+				</pluginManagement>
+			</build>
+		</profile>
+		<!-- Error prone; JDK 8 requires a different configuration than 9+ -->
 		<profile>
 			<id>errorprone_java_9_and_up</id>
 			<activation>
-				<jdk>[9,)</jdk>
+				<jdk>[9,]</jdk>
```
Comment (elefeint) [inline pom.xml:425]: Should this be 9 through 15, or 9 through latest? As is, both `errorprone_java_9_and_up` and `jdk_16_and_up` will fire on 16.

File: pom.xml [hunk 2]
```diff
@@ -431,35 +470,28 @@
 			<activation>
 				<jdk>1.8</jdk>
 			</activation>
+			<properties>
+				<javac.version>9+181-r4173-1</javac.version>
```
Comment (elefeint) [inline pom.xml:474]: uh this is oddly specific. Do we have to do this?

**Review by elefeint — CHANGES_REQUESTED** [review]
At what point should we be getting errorprone to fix issues instead of working around them? Requiring a specific java build seems a bit much.

File: pom.xml (hunk 2 above)
Comment (meltsufin) [reply pom.xml:474]: I believe this was in the errorprone docs.

File: pom.xml (hunk 1 above)
Comment (meltsufin) [reply pom.xml:425]: Good point. I was just blindly backporting from the new repo.

Comment (meltsufin) [reply pom.xml:425]: Actually, I think this is intentional. The two profiles combine for 16.

**Review by elefeint — APPROVED** [review]

**Pull request merged**